ServerCore/Pages/PuzzleApiController.cs

```diff
@@ -45,16 +46,29 @@ public async Task<ActionResult<bool>> GetValidateTeamPasswordAsync(string teamPa
         }
 
         /// <summary>
-        /// Allows external puzzles to validate a team password
+        /// Allows external puzzles to get information pertaining to how to identify that team in email
         /// </summary>
-        /// <param name="teamPassword">Potential team password</param>
-        /// <returns>True if the password belongs to a team, false otherwise</returns>
+        /// <param name="eventId">ID of the event</param>
+        /// <param name="puzzleId">ID of the puzzle</param>
+        /// <param name="teamPassword">Team password</param>
+        /// <returns>Identifying information for that team/puzzle</returns>
         [HttpGet]
+        [EnableCors("PuzzleApi")]

```

Comment:
It might be good to put this on the class so we don't have to remember it on every method